portal/test/utils/watch/pollings/analyzeEvmWithdrawalPolling.test.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  MessageDirection,
+  MessageStatus,
+  type ToEvmWithdrawOperation,
+} from 'types/tunnel'
+import { hemiSepolia, sepolia } from 'viem/chains'
+import { describe, expect, it } from 'vitest'
+import {
+  analyzeEvmWithdrawalPolling,
+  EvmWithdrawalPriority,
+} from 'workers/pollings/analyzeEvmWithdrawalPolling'
+
+const getSeconds = (seconds: number) => seconds * 1000
+const getMinutes = (minutes: number) => getSeconds(minutes * 60)
+
+const createWithdrawal = (
+  overrides: Partial<ToEvmWithdrawOperation> = {},
+): ToEvmWithdrawOperation =>
+  // @ts-expect-error Only adding the minimum required properties for testing
+  ({
+    claimTxHash: '0x456',
+    direction: MessageDirection.L2_TO_L1,
+    l1ChainId: sepolia.id,
+    l2ChainId: hemiSepolia.id,
+    proveTxHash: '0x123',
+    status: MessageStatus.RELAYED,
+    timestamp: 123456,
+    transactionHash: '0xabc',
+    ...overrides,
+  })
```

Comment:
For tests, I like to use `zeroHash` and `zeroAddress` from `viem` (Assuming that the fact that they are the same value do not affect the test)

```suggestion
    claimTxHash: zeroHash,
    direction: MessageDirection.L2_TO_L1,
    l1ChainId: sepolia.id,
    l2ChainId: hemiSepolia.id,
    proveTxHash: zeroHash,
    status: MessageStatus.RELAYED,
    timestamp: 123456,
    transactionHash: zeroHash,
    ...overrides,
  })
```

Reply:
TIL! Improved here 0276ecd397cf403a93720ddf2ec5b60b4e2349cc. Thanks!